integration-tests/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>webservices.rest</artifactId>
-		<version>2.24.0</version>
+		<version>2.25.0-SNAPSHOT</version>
```

Comment:
We do not need this change because the version is already as expected in the master branch: https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/integration-tests/pom.xml#L6

Reply:
@dkayiwa  This came as a result of running git pull --rebase upstream master. And I seem not to understand clearly what you mean. should I roll back to 2.24.0?

Reply:
That was simply because you do not have the latest upstream changes. Can you cross check if the upstream master branch has 2.24.0 or 2.25.0-SNAPSHOT?

Reply:
upstream master has 2.25.0-SNAPSHOT as version

Reply:
upstream master has 2.25.0-SNAPSHOT as version  which is what I have here.

Reply:
If the upstream version is already 2.25.0-SNAPSHOT, then it should not be among your changes.

Reply:
Ohh I see what you mean. How do I resolve this. It surely came as a result of me trying to squash some commit. and force push

Reply:
Ensure that your local master is up to date with the up stream master